src/Fixer/PhpUnit/PhpUnitAttributesFixer.php

```diff
@@ -458,18 +458,27 @@ private static function fixRequires(Tokens $tokens, int $index, Annotation $anno
         } elseif ('RequiresPhp' === $attributeName && isset($matches[3])) {
             $attributeTokens = [self::createEscapedStringToken($matches[2].' '.$matches[3])];
         } else {
-            $attributeTokens = [self::createEscapedStringToken($matches[2])];
+            $attributeTokens = [self::createEscapedStringToken(self::fixVersionConstraint($matches[2]))];
         }
 
         if (isset($matches[3]) && 'RequiresPhp' !== $attributeName) {
             $attributeTokens[] = new Token(',');
             $attributeTokens[] = new Token([T_WHITESPACE, ' ']);
-            $attributeTokens[] = self::createEscapedStringToken($matches[3]);
+            $attributeTokens[] = self::createEscapedStringToken(self::fixVersionConstraint($matches[3]));
         }
 
         return self::createAttributeTokens($tokens, $index, $attributeName, ...$attributeTokens);
     }
 
+    private static function fixVersionConstraint(string $version): string
```

Comment:
@sebastianbergmann is this how version constraint from annotation should be converted for attributes? The argument here is what is in annotation (e.g. `@requires PHP 8.2`) and the returned value is what will be in the attribute (e.g. `#[RequiresPhp('>= 8.2')]`.